src/SelfAssessment/endScreens/Emergency.js

```diff
@@ -14,7 +14,7 @@ export const Emergency = () => {
   let { t } = useTranslation()
 
   // TODO: This would need to be localized per country
```

Comment:
I'm creating a ticket on trello with a task for this TODO How do you feel about removing it in this PR?

Reply:
Sounds good!